main.tf

```diff
@@ -112,9 +112,9 @@ module "configure_project" {
   watsonx_project_description = var.watsonx_project_description
```

Comment:
Can you please change  `watsonx_project_description` to `project_description` ?

Reply:
Done.